cdk/package.json

```diff
@@ -13,13 +13,13 @@
     "lint": "eslint lib/** bin/** --ext .ts --no-error-on-unmatched-pattern"
   },
   "devDependencies": {
-    "@aws-cdk/assert": "~1.74.0",
+    "@aws-cdk/assert": "1.86.0",
     "@guardian/eslint-config-typescript": "^0.4.1",
     "@types/jest": "^26.0.20",
     "@types/node": "10.17.27",
     "@typescript-eslint/eslint-plugin": "^4.14.0",
     "@typescript-eslint/parser": "^4.14.0",
-    "aws-cdk": "~1.74.0",
+    "aws-cdk": "1.86.0",
```

Comment:
These version bumps are required to keep us in sync after https://github.com/guardian/cdk/pull/191.